From: Sven Eckelmann <sven@narfation.org>
Date: Sat, 3 Dec 2016 14:30:05 +0100
Subject: batctl: upgrade package to latest release 2016.5

 * reimplement traceroute/ping commands in userspace without debugfs
 * switch interface manipulation from (legacy) sysfs to rtnetlink
 * coding style cleanups

Signed-off-by: Sven Eckelmann <sven@narfation.org>

Origin: backport, https://github.com/openwrt-routing/packages/commit/c425a4d8a6cc5fa29ab13d585e7115ded69387ed

diff --git a/batctl/Makefile b/batctl/Makefile
index 7ae81b57c8850c85008bfc3c704677077d2be59d..bf305a24b522f118b3f52784dfd4fb28e38db351 100644
--- a/batctl/Makefile
+++ b/batctl/Makefile
@@ -9,9 +9,9 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=batctl
 
-PKG_VERSION:=2016.4
+PKG_VERSION:=2016.5
 PKG_RELEASE:=0
-PKG_MD5SUM:=0112f35b2a48d32f23a17a6f45348e63
+PKG_MD5SUM:=7b33fb47c7fa5b317e9a152a286999fc
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://downloads.open-mesh.org/batman/releases/batman-adv-$(PKG_VERSION)
@@ -25,7 +25,7 @@ define Package/batctl
   URL:=https://www.open-mesh.org/
   SECTION:=net
   CATEGORY:=Network
-  DEPENDS:=+kmod-batman-adv +libnl-tiny +libc
+  DEPENDS:=+kmod-batman-adv +libnl-tiny +libc +librt
   TITLE:=B.A.T.M.A.N. Advanced user space configuration tool batctl
   MAINTAINER:=Marek Lindner <mareklindner@neomailbox.ch>
 endef
